.github/workflows/e2e-testnet.yml

```diff
@@ -0,0 +1,45 @@
+name: E2E tests / Testnet
+
+on:
+  schedule:
+    - cron: '*/20 * * * *'
+  workflow_dispatch:
+  pull_request: # TODO: remove this trigger before merging to main
+
+jobs:
+  e2e-testnet-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./e2e
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+
+      # TODO: Temprorarily we use fixed version of tbtc.js. Ultimately, we would
+      # need to point to the latest deployed version.
+      - run: npm install --save-exact @keep-network/tbtc.js@0.19.4-ropsten.4
```

Comment:
Once https://github.com/keep-network/tbtc.js/pull/129 lands and a new package for `ropsten` is published, this line should be:
```
- run: npm install --save-exact @keep-network/tbtc.js@ropsten
```
and it should always pull the latest package for `ropsten`.

Reply:
Added. After I published the `ropsten`-tagged `tbtc.js` package, step is passing, correct package was used.
Workflow run: [link](https://github.com/keep-network/local-setup/pull/98/checks?check_run_id=3202543359) (workflow is failing, but on later stage - will be investigated)